user/views.py

```diff
@@ -71,3 +70,85 @@ def post(self, request):
         except KeyError:
             return JsonResponse({'message':'KEYERROR'}, status=400)
             
+class LikeProductView(View):
+    @Login_required
+    def post(self, request):
+        user_data = json.loads(request.body)
+
+        if not LikeProduct.objects.filter(product_id = user_data['product_id'], user_id = request.user.id).exists():
+            LikeProduct.objects.create(
+                user    = request.user,
+                product = Product.objects.get(id = user_data['product_id'])
+            ).save
+
+            return JsonResponse({'message':'ADD LIKE_PRODUCT'}, status=200)
+
+        else:
+            delete_product = LikeProduct.objects.get(product_id = user_data['product_id'], user_id = request.user.id)
+            delete_product.delete()
+
+            return JsonResponse({'message':'DELETE LIKE_PRODUCT'}, status=200)
```

Comment:
`if`문을 만족해서 return 되지 않는다면 이 코드에 도달할 수 있는 로직입니다. 따라서 else를 없애고 코드를 한 단락 앞으로 빼주세요
불필요한 `else`는 사용하지 않는 것이 보다 파이썬다운 코드입니다 :) (Pythonic rule)
```suggestion

        
        delete_product = LikeProduct.objects.get(product_id = user_data['product_id'], user_id = request.user.id)
        delete_product.delete()

        return JsonResponse({'message':'DELETE LIKE_PRODUCT'}, status=200)
```